confluent_kafka/src/confluent_kafka.c

```diff
@@ -204,7 +204,7 @@ static long KafkaError_hash (KafkaError *self) {
 	return self->code;
 }
 
-static PyTypeObject KafkaErrorType;
+PyTypeObject KafkaErrorType;
```

Comment:
Removing static makes this variable global to the program, but it is not used outside of this file so there is no point in doing so.